docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -0,0 +1,276 @@
+and set the AD admin using _Set admin_ button:
+
+![Set AD admin](../../../img/database-access/guides/azure/set-ad-admin@2x.png)
+
+Note that only one AD user can be set as an admin for the database.
+
+### Connect to database as AD admin
+
+Next, you need to connect to your database as the AD admin user.
+
+Use Azure `az` CLI utility to log in as the user that you set as the AD admin,
+fetch the access token and use it as a password when connecting to the database:
+
+<Tabs>
+<TabItem label="PostgreSQL">
+  ```bash
+  $ az login -u ad@example.com
+  $ export PGPASSWORD=`az account get-access-token --resource-type oss-rdbms --output tsv --query accessToken`
+  $ psql "host=example.postgres.database.azure.com user=ad@example.com@instance-name sslmode=require dbname=postgres"
+  ```
+</TabItem>
+<TabItem label="MySQL">
+  ```bash
+  $ az login -u ad-admin@example.com
+  $ export TOKEN=`az account get-access-token --resource-type oss-rdbms --output tsv --query accessToken`
+  $ mysql -h example.mysql.database.azure.com -P 3306 -u ad@example.com@instance-name --enable-cleartext-plugin --password=$TOKEN
+  ```
+</TabItem>
+</Tabs>
+
+Note that the database username must include `@instance-name` suffix with the
+name of the Azure database instance you're connecting to.
+
+### Create AD users
+
+Once connected to the database as AD admin, create database users for the
+service principal Teleport Database Service will be using. Use `Client ID`
+when using managed identities and `Application (client) ID` when using app
+registrations:
+
+<Tabs>
+<TabItem label="PostgreSQL">
+  ```
+  postgres=> SET aad_validate_oids_in_tenant = off;
+  SET
+  postgres=> CREATE ROLE teleport WITH LOGIN PASSWORD '11111111-2222-3333-4444-555555555555' IN ROLE azure_ad_user;
+  CREATE ROLE
+  ```
+</TabItem>
+<TabItem label="MySQL">
+  ```
+  mysql> SET aad_auth_validate_oids_in_tenant = OFF;
+  mysql> CREATE AADUSER 'teleport' IDENTIFIED BY '11111111-2222-3333-4444-555555555555';
+  Query OK, 0 rows affected (0.92 sec)
+  ```
+</TabItem>
+</Tabs>
+
+You can create multiple database users for the same service principal.
+
+## Step 4/4. Connect
+
+Log into your Teleport cluster, your Azure database should appear in the list of
+available databases:
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name     Description         Labels
+# -------- ------------------- -------
+# azure-db                     env=dev
+```
+
+Fetch short-lived client certificate for it using `tsh db login` command:
+
+```code
+$ tsh db login --db-user=teleport azure-db
+```
+
+<Admonition type="tip">
+  You can be logged into multiple databases simultaneously.
+</Admonition>
+
+Now connect to the database:
+
+```code
+$ tsh db connect azure-db
+```
+
+<Admonition type="note">
+  The appropriate database command-line client (`psql`, `mysql`) should be
+  available in PATH in order to be able to connect.
```

Comment:
Does this refer to the end-user's machine or the VM running the Database Service?